rotkehlchen/db/addressbook.py

```diff
@@ -92,13 +100,18 @@ def add_addressbook_entries(
             self,
             book_type: AddressbookType,
             entries: list[AddressbookEntry],
+            cursor: Optional['DBCursor'] = None,
+            exists_ok: bool = False,
     ) -> None:
         """
         Add every entry of entries to the address book table.
         If blockchain is None then make sure that the same address doesn't appear in combination
         with other blockchain values.
+
+        If cursor is given then it is used, otherwise a cursor that corresponds to book_type
```

Comment:
We **never** do that. Anywhere in the code. At all. Ever. Didn't you wonder why?

Because it does not work. It does not scale. You either pass a cursor always or you never pass a cursor. Delete this optionality hack.